README.md

```diff
@@ -146,18 +156,32 @@ $ helm dependency update
 
 After doing this, helm will download the `spring-boot-example-app` chart from the repository `http://snowdrop.github.io/helm` and will copy the Chart tallball `spring-boot-example-app-0.0.3.tgz` at `my-custom-chart/charts/`.
```

Comment:
`After doing this, helm will download the `spring-boot-example-app` chart from the repository ...`. Are you sure ? This is not correct from my point of view to say that as first `helm install` is gonna to deploy using the chart the K8S manifests on the cluster. When you use OpenShift, it is needed to deploy a BuildConfig and Build resources which are responsible on OpenShift to trigger the creation of a pod doing the S2I build. To be reviewed

Reply:
This sentence does not mention OpenShift and explains what the command `helm dependency update` does. 
What happens with OpenShift or Kubernetes, it's described in a previous section.